nixos/modules/services/x11/dunst.nix

```diff
@@ -0,0 +1,134 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+   cfg = config.services.dunst;
+   reservedSections = [
+     "global" "experimental" "frame" "shortcuts"
+     "urgency_low" "urgency_normal" "urgency_critical"
+   ];
+in {
+
+options.services.dunst = {
+
+  enable = mkEnableOption "the dunst notifications daemon";
+
+  iconDirs = mkOption {
+    type = with types; listOf path;
+    default = [];
+    example = literalExample ''
+      [ "''${pkgs.gnome3.adwaita-icon-theme}/share/icons/Adwaita/48x48" ]
+    '';
+    description = ''
+      Paths to icon folders.
+    '';
+  };
+
+  extraCliOptions = mkOption {
```

Comment:
I suggest renaming this to either `extraArgs` or `extraOptions` since this is a more common name for this